docs/modules/ROOT/pages/how-tos/testing/integration/third-parties/testing-farm.adoc

```diff
@@ -0,0 +1,55 @@
+= Testing Farm
+
+In this guide, you'll learn how to xref:/how-tos/testing/integration/adding.adoc[add a custom integration test] in {ProductName} that uses link:https://docs.testing-farm.io/[Testing Farm] as a third-party backend for test execution.
+
+.Prerequisites
+
+. You have xref:/how-tos/creating.adoc[created an application] in {ProductName}
```

Comment:
Users would need to create an application and a component?

Reply:
True!